indico/modules/events/timetable/client/js/utils.ts

```diff
@@ -59,6 +63,14 @@ function gcd(a: number, b: number) {
   return a;
 }
 
+export const getEntryURL = (entry: {type: EntryType; objId: number | string}, eventId: number) =>
```

Comment:
Just a small nitpick, for "real" functions I prefer using the `=> { return ... }` syntax instead of the `=> expression` syntax. Otherwise adding any console logs for debugging is a pain because it's no longer just adding an extra line but changing the syntax of the function.